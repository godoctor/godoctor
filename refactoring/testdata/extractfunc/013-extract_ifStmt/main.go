// <<<<< extract,9,2,11,2,foo,pass

package main

import "fmt"

func main() {
	i := 2
	if i%2 == 0 {
		fmt.Println("i is even")
	}
	fmt.Println("end of main function")
}
