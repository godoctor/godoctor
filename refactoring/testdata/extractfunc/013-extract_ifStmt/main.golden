// <<<<< extract,9,2,11,2,foo,pass

package main

import "fmt"

func main() {
	i := 2
	foo(i)
	fmt.Println("end of main function")
}
func foo(i int) {
	if i%2 == 0 {
		fmt.Println("i is even")
	}
}
